runxtb.sh

```diff
@@ -362,22 +362,29 @@ load_xtb_modules ()
   ( command -v module &>> "$tmpfile" ) || fatal "Command 'module' not available."
   # Try to load the modules, but trap the output in the temporary file.
   # Exit if that fails (On RWTH cluster the exit status of modules is always 0).
-  module load "${load_modules[*]}" &>> "$tmpfile" || fatal "Failed to load modules."
+  # The new Lmod module system throws errors when loading all modules sequentially in one
+  # command, thus they are now loaded sequentially.
+  # Also checks if on RWTH cluster as the intel toolchain as to be unloaded before the foss
+  # toolchain can be loaded without throwing errors.
+  if [[ $(uname -n) == *"rwth"* ]]; then
+    module unload intel &>> "$tmpfile"
+  fi
+
+  for mod in "${load_modules[@]}" ; do
+    module load "${mod}" &>> "$tmpfile" || fatal "Failed to load module."
+  done
   # Remove colourcodes with sed:
   # https://www.commandlinefu.com/commands/view/12043/remove-color-special-escape-ansi-codes-from-text-with-sed
   sed -i 's,\x1B\[[0-9;]*[a-zA-Z],,g' "$tmpfile"
   # Check whether then modules were loaded ok
   local check_module
```

Comment:
that seems to be not used anymore

Reply:
I believe this remains necessary, since this function is called when running locally, see [line 949](https://github.com/polyluxus/runxtb.bash/blob/7518ef7ee114403f690e95a00dba981a1cb2111d/runxtb.sh#L949)